### PR TITLE
add table printer / BDD step for creating persistent resource / asset fixtures

### PR DESCRIPTION
**Pull request by erickloss**
for now, only ImageInterface is supported. Other asset types must also be implemented.

also fixes: #13 #9 

I tried to keep the Neos specific logic outside of the package, but added a hook / README.
When run with Neos/Flow After importing asset fixtures, `resource:publish` must be called somehow with the SUT context.

- [x] better handling of fixture base paths (currently, the absolute file path is printed)
- [x] finalize documentation about how to create /use persistent resource / asset fixtures
- [x] add handling for more than `ImageInterface` typed node properties (other Neos.Media models extending Asset)

please give feedback <3 see comments in the PR

cheers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -238,69 +235,69 @@ main job) and all related services to the `variables` section of the test job.
...
+        - cd /app && rm -Rf e2e-results && mkdir e2e-results && bin/behat     --format junit --out e2e-results       --format pretty --out std       -c Packages/Application/PACKAGEKEY/Tests/Behavior/behat.yml.dist
+        - cp -R /app/e2e-results $CI_PROJECT_DIR/e2e-results
+        - cp -R /app/Web/styleguide $CI_PROJECT_DIR/styleguide
+    artifacts:
+        expire_in: 4 weeks
+        paths:
+            - e2e-results
+            - styleguide
+        reports:
+            junit: e2e-results/behat.xml
```
Comment (skurfuerst) [inline README.md:294]: hier war der autoformatter etwas wild, oder? ;)

File: README.md [hunk 2]
```diff
@@ -604,27 +608,77 @@ This is ready to be pasted into a test case like the following:
...
+You probably want to store you asset fixtures near your feature files.
+
+# TODO explain how to set fixture base path
+
+```php
+    // ... Step Generator Command Controller 
+
+    public function homepageCommand()
+    {
+        $fixtureBasePath = TODO !!!! FIX DOC !!!!
```
Comment (skurfuerst) [inline README.md:646]: TODO?

**Review by skurfuerst — COMMENTED** [review]
only nitpicks :)

**Comment by erickloss** [comment]
This change should be downwards compatible. (pls recheck briefly)

Followup Ticket: #19

**Pull request merged**